sros2/sros2/policy/__init__.py

```diff
@@ -16,45 +16,47 @@
 
 from lxml import etree
 
-import pkg_resources
+try:
+    import importlib.resources as importlib_resources
+except ModuleNotFoundError:
+    # (clalancette): We have to ignore the type, otherwise mypy throws a warning.
```

Comment:
```suggestion
    # This fallback is necessary when importlib-resources is installed from pip
    # (clalancette): We have to ignore the type, otherwise mypy throws a warning.
```

Reply:
Makes sense; see b3a53fd.